cityofaustin/atd-moped
# Add Project Tags feature

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/10044

## Testing
**URL to test:** 
https://10044-tags--atd-moped-main.netlify.app/moped/projects/161

**Steps to test:**

Navigate to a project's summary page
Add Tags
Check that after adding a tag, the tag is no longer available as an option in the dropdown
Delete Tag



---
#### Ship list
